src/main/java/com/uid2/shared/optout/OptOutCloudSync.java

```diff
@@ -337,11 +337,8 @@ private void localListFiles(ICloudStorage fsLocal, String dirToScan, String file
             } else if (OptOutUtils.getFileTimestamp(f) == null) {
                 LOGGER.warn("Unrecognized timestamp, unknown file " + f);
             }
-            if (Utils.IsWindows && f.indexOf(':') > 0) {
```

Comment:
What is the reason for removing windows specific stuff?

Reply:
Actually I have to remove this to make it run on windows.